apps/api/plane/db/models/description.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.utils.html import strip_tags
+from .workspace import WorkspaceBaseModel
+
+
+class Description(WorkspaceBaseModel):
+
+
+    description_json = models.JSONField(default=dict, blank=True)
+    description_html = models.TextField(blank=True, default="<p></p>")
+    description_binary = models.BinaryField(null=True)
+    description_stripped = models.TextField(blank=True, null=True)
+
+    class Meta:
```

Comment:
@NarayanBavisetti Check if we can add any indexes that will help with query performance.